cdap-watchdog/src/test/java/co/cask/cdap/logging/pipeline/kafka/KafkaLogProcessorPipelineTest.java

```diff
@@ -78,6 +78,7 @@
 
 import java.io.File;
 import java.io.IOException;
+
```

Comment:
Why there are so many extra lines introduced?

Reply:
Hmm seems like rebase to develop introduced them. Fixed